portal-api/src/v1/gef/controllers/clone-gef-deal.controller.js

```diff
@@ -161,7 +164,7 @@ exports.clone = async (req, res) => {
   if (response.status === 200) {
     const { newDealId } = response;
     // clone the corresponding facilities
-    await cloneFacilities(existingDealId, newDealId);
+    await cloneFacilities(existingDealId, newDealId, req.user);
```

Comment:
sorry to be a pain but can we create the audit details inside `exports.clone` and pass it in please?

Reply:
(same with cloneDeals)